src/transport/MessageCounter.cpp

```diff
@@ -29,7 +29,7 @@ namespace chip {
 
 void GlobalUnencryptedMessageCounter::Init()
 {
-    mValue = Crypto::GetRandU32();
+    mLastUsedValue = Crypto::GetRandU32();
```

Comment:
I noticed this after merge, but the spec has *all* counters initializing to: `(0 to 2^28) + 1`.